src/app/Pipes/summary.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'summary',
+})
+export class SummaryPipe implements PipeTransform {
+  transform(value: string, limit: number, ...args: unknown[]): unknown {
+    limit = limit ? limit : 30;
+    if (value.length <= 25) {
```

Comment:
return value.length <= 25 ? value : `${value.substring(0, limit)}...`;
can use `slice` function here

Reply:
it works. pls be flexible next time. @DienMinh 
![image](https://user-images.githubusercontent.com/47804324/164279831-3b432f2f-0c41-4b26-b5d2-70c72bdd2f4d.png)